CONTRIBUTING.md

```diff
@@ -103,26 +103,46 @@ You are ready to go 🎉.
 
 ### Common commands
 
+> [!TIP]
+> To run these commands you don't need to activate virtualenv via `poetry env activate`
```

Comment:
I would add info that if you are in devcontainer you don't need to active env 😄

Reply:
Is that sentence correct? You added poe to `pyproject`, which will install the `poe` to the env which is active when you call `poetry install` right? Then to use it, you need to have active virtual env (assuming that you're using virutalenv).  Additionally, the poe's commands, calls django commands, which would require the virtualenv. Do I miss something? 🤔

Reply:
@korycins you were right, it worked _for me_ because I've installed `poe` as their docs suggested via `pipx` globally 😅

if it's installed only for Saleor, then this won't work, I've removed this tip.